ticdc/manage-ticdc.md

```diff
@@ -742,9 +742,11 @@ sync-ddl = true
 2. 开启环形同步的数据表名字需要符合正则表达式 `^[a-zA-Z0-9_]+$`。
 3. 在创建环形同步任务前，开启环形复制的数据表必须已创建完毕。
 4. 开启环形复制后，不能创建一个会被环形同步任务同步的表。
-5. 如果想在线 DDL，需要确保以下两点：
-    1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
-    2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+5. 在多集群同时写入时，为了避免业务出错，请避免执行 DDL，比如 `ADD COLUMN`/`DROP COLUMN` 等。 
```

Comment:
It's better to provide a whitelist of DDL. e.g. `ADD INDEX`, `DROP INDEX`.

Reply:
Could you add your inputs using the review suggestion?


\`\`\` suggestion

\`\`\`

Reply:
Some DDLs such as `ADD INDEX` could work here, but we don't have enough tests for this feature, IMHO we can leave the enforcing ban of DDL here